mne/time_frequency/tests/test_spectrum.py

```diff
@@ -141,18 +142,21 @@ def test_spectrum_io(inst, tmp_path, request, evoked):
     assert orig == loaded
 
 
-def test_spectrum_copy(raw):
+def test_spectrum_copy(raw_spectrum):
```

Comment:
Although I'd have preferred adding the fixture in its own PR, I decided it was expedient to just go with it, so I've used it in the other spectrum tests too.